docs/cmdlets/Get-OutputBinding.md

```diff
@@ -13,7 +13,7 @@ Gets the hashtable of the output bindings set so far.
 ## SYNTAX
 
 ```
-Get-OutputBinding [[-Name] <String[]>] [-Purge] [<CommonParameters>]
+Get-OutputBinding [-Name <String>] [-Purge] [<CommonParameters>]

```

Comment:
Can we go back to this being positional?

Reply:
Sure, will make the change.

Reply:
Fixed. It turns out parameter is made positional even without specifying `Position` for advanced function.